scripts/addFromConfig.py

```diff
@@ -162,9 +156,9 @@ def _fileTest(filename):
         raise ValueError('Specified section(s): "{0}" is repeated!'.format(seen_twice))
 
 
-def addStuff(cfg, options):
+def addStuff(cfg, options, instrument):
     # setup the db
-    dbu = DButils.DButils(options.mission)
+    dbu = DButils.DButils(options.mission, instrument)
```

Comment:
As written, this passes instrument as the `db_var` argument to DButils, which does nothing, and this is the only use of `instrument`...is there a reason here?

Reply:
That's coming from our LANL code.   I think you need "instrument" in `addStuff()`, but hopefully no longer in DButils.DButils.  `instrument` is everywhere here, and now we have a new database and use mission for each `instrument`.  I am trying to get rid of `instrument` the way it is currently done.   (In the old ways, 1 mission had more than 200 instruments, I prefer 200 databases which make search easier.) 

Actually in `addFromConfig`, I don't think I can add more than 1 satellite right now things are intertwine with `instrument_id`

`addFromConfig` will need more work for sure.  It's a bit challenging to merge your way to link things and our way here at LANL.

Reply:
But the only reason addStuff needs the instrument argument is to pass it to DButils, which doesn't use it. The calls to addStuff with the instrument argument were added in this PR...if it's all left out, it works just fine.